index.html

```diff
@@ -5,330 +5,360 @@
+                            </a>
                         </li>
                     </ul>
-                </div>
-                <div class="resume">
-                    <a href="#" class=" resumetext">
-                        <p>Get My Resume</p>
-                    </a>
-                </div>
-            </div>
-            <div class="container6">
-                <div class="languages">
-                    <a href="#" class="langcontainer">
-                        <h4>Languages</h4> <img class="arrow3" src="./Images/right-arrow.png" alt="">
-                    </a>
-                </div>
-
-                <ul class="last">
-                    <li class="js">
-                        <a href="#"> <img class="EJS" src="./Images/Frame9.png" alt="#">
+                    <div class="frameworks">
+                        <a href="#" class="framecontainer">
+                            <h4>Frameworks</h4>
                         </a>
-                    </li>
-                    <li class="html">
-                        <a href="#"> <img class="EH" src="./Images/Frame10.png" alt="#">
-                        </a>
-                    </li>
-                    <li class="css">
-                        <a href="#"> <img class="CS" src="./Images/Frame11.png" alt="#">
+                    </div>
+                    <div class="line">
+                        <img class="subs" src="./Images/subtract.png" alt="">
+                    </div>
+                    <div class="skills">
+                        <a href="#" class="skillscontainer">
+                            <h4>Skills</h4>
                         </a>
-                    </li>
-                </ul>
-                <div class="frameworks">
-                    <a href="#" class="framecontainer">
-                        <h4>Frameworks</h4> <img class="arrow" src="./Images/right-arrow.png" alt="">
-                    </a>
+                    </div>
+                    <div class="line">
+                        <img class="subs" src="./Images/subtract.png" alt="">
+                    </div>
                 </div>
-                <div class="line">
-                    <img class="subs" src="./Images/subtract.png" alt="">
+            </div>
+            <div class="cont7">
+                <div class="contact">
+                    <h3>Contact Me</h3>
                 </div>
-                <div class="skills">
-                    <a href="#" class="skillscontainer">
-                        <h4>Skills</h4> <img class="arrow2" src="./Images/right-arrow.png" alt="">
-                    </a>
+                <div class="context">
+                    <p>If you have an application you <br> are interested in developing, a <br> feature that you need built or a <br> project that needs coding. I’d <br> love to help with it </p>
+                </div>
+                <div class="forms">
```

Comment:
- [ ] _[OPTIONAL]_ this element should be wrapped in its own `section` tag.
- [ ] On the desktop screen, your contact section has unwanted margins on both sides filled with gray-colored space, kindly inspect this issue and make sure to follow the Figma design as much as possible.
![image](https://user-images.githubusercontent.com/33767951/140237644-2ec7d1d4-1af9-4f03-bb6b-cb5f652d1688.png)